go.mod

```diff
@@ -28,7 +28,7 @@ require (
 )
 
 replace (
-	github.com/gardener/machine-controller-manager => github.com/gardener/machine-controller-manager v0.33.1-0.20200828071210-90f8b67cc5e6
+	github.com/gardener/machine-controller-manager => github.com/gardener/machine-controller-manager v0.39.0
```

Comment:
MCM v0.39.0 is being vendored under the `require` section. I do not see a reason to replace it with the same version under the `replace` section of this `go.mod`. You can remove this line.